packages/documentation/docs/controls/_select_styleguide.md

```diff
@@ -0,0 +1,66 @@
+Selects provide users to make a single or multi selection from a filterable list. With the editable variant, users can also add new options to the list.
+
+![Overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3054-593&t=6gU2IFkPWR943af8-4)
```

Comment:
When I open this link, input field illustration is opened. Should link to Select: https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3647-6332&t=DtCmoFcLwhf7ke3S-4